spec/models/repository/git_spec.rb

```diff
@@ -87,7 +87,8 @@
   describe 'with an actual repository' do
     with_git_repository do |repo_dir|
       let(:url)      { repo_dir }
-      let(:instance) { FactoryGirl.create(:repository_git, path_encoding: encoding, url: url) }
+      let(:instance) { FactoryGirl.create(:repository_git, path_encoding: encoding,
+                                          url: url, root_url: url) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 91, 68 should be on its own line.